dependencies.yaml

```
@@ -140,10 +169,13 @@ dependencies:
           - cupy>=12.0.0
       - output_types: requirements
```

Due to `--extra-index-url` reasons for ptxcompiler/cubinlinker, cudf wasn't able to combine these. Luckily, cuxfilter can combine them.
```suggestion
      - output_types: [requirements, pyproject]
```

---

fixed it